Rakefile

```diff
@@ -67,67 +67,55 @@ RSpec::Core::RakeTask.new("spec:tools") do |t|
   t.pattern = "spec/tools/**/*_spec.rb"
 end
 
-def configure_machinery
-  Packaging.configuration do |conf|
-    conf.obs_api = "https://api.opensuse.org"
-    conf.obs_project = "systemsmanagement:machinery"
-    conf.package_name = "machinery"
-    conf.obs_target = "openSUSE_13.2"
-
-    #lets ignore license check for now
-    conf.skip_license_check << /.*/
-  end
-end
-
-def configure_machinery_helper
-  Packaging.configuration do |conf|
-    conf.obs_api = "https://api.opensuse.org"
-    conf.obs_project = "systemsmanagement:machinery:helper"
-    conf.package_name = "machinery-helper-x86_64"
-    conf.obs_target = "SLE_12"
-
-    #lets ignore license check for now
-    conf.skip_license_check << /.*/
-  end
+Packaging.configuration do |conf|
+  conf.obs_api = "https://api.opensuse.org"
+  conf.obs_project = "systemsmanagement:machinery"
+  conf.package_name = "machinery"
+  conf.obs_target = "openSUSE_13.2"
+  conf.version = Machinery::VERSION
+
+  #lets ignore license check for now
+  conf.skip_license_check << /.*/
 end
 
 namespace :man_pages do
   desc 'Build man page(s)'
   task :build do
-    if Dir.exist?("man")
```

Comment:
The helper didn't had a man page directory before and the method should have worked on both.